src/utilities/utilities.js

```diff
@@ -169,3 +170,49 @@ export const handleSelfFocus = e => {
     e.currentTarget.focus();
   }
 };
+
+export const isFullyVisible = (slideIndex, config) => {
+  const {
+    currentSlide,
+    cellSpacing,
+    slideCount,
+    slideWidth,
+    frameWidth,
+    wrapAround
+  } = config;
+
+  // Slide width can't be 0
+  const fullSlideWidth = slideWidth || 1;
+  // Calculate offset without cellSpacing
+  const offsetWidth =
+    getAlignmentOffset(currentSlide, config) + cellSpacing * currentSlide;
```

Comment:
Adding back the cellSpacing offset fixed the bug where visible slides were being tagged as `inert` when cellSpacing was greater than 0.